docs/dev/w3c-webdriver-capabilities.md

```diff
@@ -155,35 +155,62 @@ Browser Options classes are used to manage both browser specific functionality a
 
 Here is a comparison of deprecated Java code that needs to be replaced with recommended code:
 
-```java title="Deprecated Code"
+<Tabs
+  defaultValue="Deprecated Code"
+  values={[
+    {label: 'Deprecated Code', value: 'Deprecated Code'},
+    {label: 'Recommended Code (Selenium 4)', value: 'Recommended Code (Selenium 4)'},
+  ]}>
+
+<TabItem value="Deprecated Code">
+
+```java
 DesiredCapabilities caps = new DesiredCapabilities.firefox();
 caps.setCapability("platform", "Windows 10");
 caps.setCapability("version", "latest");
 ```
 
-:::warning
+:::note
 This deprecated code is no longer available in Selenium 4.0+.
 :::
 
-```java title="Recommended Code (Selenium 4)"
+</TabItem>
+<TabItem value="Recommended Code (Selenium 4)">
+
+```java
 FirefoxOptions options = new FirefoxOptions();
 options.setPlatformName("Windows 10");
 options.setBrowserVersion("latest");
 ```
 
+</TabItem>
+</Tabs>
+
 #### W3C Sauce Labs Options
 
 Sauce Labs specific capabilities used to be able to go in the top-level Capabilities. To be W3C-compliant, users must now put Sauce Labs configurations inside a `sauce:options` key.
 
-```java title="Deprecated Code"
+<Tabs
+  defaultValue="Deprecated Code"
+  values={[
+    {label: 'Deprecated Code', value: 'Deprecated Code'},
+    {label: 'Recommended Code (Selenium 4)', value: 'Recommended Code (Selenium 4)'},
+  ]}>
+
+<TabItem value="Deprecated Code">
+
+```java
 caps.setCapability("username", "someuser");
```

Comment:
Hi @diemol , I heard that having `username`/`accessKey` caps within `sauce:options` might be a legacy capability. Can you confirm if this is true?

Reply:
👋 @kimsauce 
I do not know. Last thing I was aware is that both are actively supported. In the past, we were discouraging the user of `username` and `accessKey` because the raw values were being shown in the metadata tab when checking the test results. This has been fixed since a while ago, so it is ok to use them.

Reply:
great, thank you I will keep them!